.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  env: {
+    browser: true,
```

Comment:
None of the code you have here runs in the browser. this shouldn't be set to true.

Reply:
I think this is just from the CLI set-up config. I think all/most of these changes are going to be trashed when I get back to trying to get this to work.